includes/class-newspack-popups-model.php

```diff
@@ -890,16 +892,12 @@ public static function container_style( $popup ) {
 	}
 
 	/**
-	 * Endpoint to report Pop-up data.
+	 * Endpoint to handle Pop-up data.
 	 *
 	 * @return string Endpoint URL.
 	 */
 	public static function get_reader_endpoint() {
-		return str_replace(
```

Comment:
To be honest I don't recall why we had this replacement to the double-slash scheme, but there probably was a good reason. Any particular reason why you removed?

Reply:
I've removed it because it does not have any effect on sites served over HTTPS, this code would only change a URL with `http://`. It just looked weird and I'm not sure AMP would even talk to an endpoint not served over HTTPS. So I don't see a good reason for this replacement, but maybe I'm missing something here.

Reply:
> I've removed it because it does not have any effect on sites served over HTTPS, this code would only change a URL with `http://`. It just looked weird and I'm not sure AMP would even talk to an endpoint not served over HTTPS. So I don't see a good reason for this replacement, but maybe I'm missing something here.

Yeah, I reviewed and couldn't find a good reason. Good call.